src/client/src/routes/ActivateUserPage/components/activate-user.js

```diff
@@ -1,15 +1,31 @@
-import React, { useState } from 'react';
+import React from 'react';
 import axios from 'axios';
-import { Link } from 'react-router-dom';
 
-function ActivateUser(props) {
-    try {
-        console.log(window.location.href);
-        //const res = await axios.post('http://localhost:5000/users/activation', user);
-    }
-    catch (err) {
-        console.log(err);
+function ActivateUser (props) {
+    const onActivate = async (e) => {
+        try {
```

Comment:
why nested try blocks?

Reply:
You're right, broke it down to two separate functions